website/docs/utilities/edit.md

```diff
@@ -1,164 +1,69 @@
-	{{if (not $embedPresent)}}{{$embed = $.nil}}{{end}}
-
-	{{if not $error}}
-		{{if or $content (ne (print $embed) "<nil>")}}
-			{{editMessage $channel.ID $id (complexMessageEdit "content" $content "embed" $embed)}}
-			Done :+1:
-		{{else}}
-			{{$error = "Content and embed cannot be null at the same time."}}
-		{{end}}
-	{{end}}
-{{end}}
-
-{{if $error}}
-	{{$helpMsg.Set "description" (print "**Error** - `" $error  "`\n" ($helpMsg.Get "description"))}}
-	{{sendMessage nil (cembed $helpMsg)}}
-{{end}}
-{{deleteResponse 5}}
+## Usage
+
+`-edit [channel] <msg> <flags...>` - Edits the message provided.
+
+The usage of this command is very similar to that of the `SimpleEmbed` built-in command. You specify a channel (defaulting to the current channel), a message, and some flags to specify which fields to edit.
+
+### Available Flags
+
+#### Flags to edit message
+
+| Flag                 | Description                                          | Example                                            |
+| -------------------- | ---------------------------------------------------- | -------------------------------------------------- |
+| `-content <content>` | Sets the content of the message                      | `-content hello world`                             |
+| `-title <title>`     | Sets the title of the embed in the message           | `-title new title`                                 |
+| `-desc <desc>`       | Sets the description of the embed in the message     | `-desc my desc`                                    |
+| `-image <url>`       | Sets the image URL of the embed in the message       | `-image https://i.imgur.com/vfbFEif.jpeg`          |
+| `-thumbnail <url>`   | Sets the thumbnail URL of the embed in the message   | `-thumbnail https://i.imgur.com/vfbFEif.jpeg`      |
+| `-url <url>`         | Sets the URL of the embed in the message             | `-url https://www.youtube.com/watch?v=oHg5SJYRHA0` |
+| `-author <name>`     | Sets the author name of the embed in the message     | `-author foo`                                      |
+| `-authoricon <url>`  | Sets the author icon of the embed in the message     | `-authoricon https://i.imgur.com/vfbFEif.jpeg`     |
+| `-footer <text>`     | Sets the footer text of the embed in the message     | `-footer bar baz`                                  |
+| `-footericon <url>`  | Sets the footer icon URL of the embed in the message | `-footericon https://i.imgur.com/vfbFEif.jpeg`     |
+| `-color <hex>`       | Sets the color of the embed in the message           | `-color FF0000`                                    |
+
+#### Flags to modify edit behaviour
+
+- `-force` - Completely overwrites the existing message. By default, the changes are merged into the existing message - for example, `-edit 123 -color FF0000` would just edit the color and keep the embed. However, by supplying the `-force` flag, `-edit 123 -color FF0000` would remove any other parts of the embed, keeping only the new color.
```

Comment:
```suggestion
- `-force` - Completely overwrites the existing message. By default, the changes are merged into the existing message - for example, `-edit 123 -color FF0000` would just edit the color and keep the embed. However, by supplying the `-force` flag: `-edit 123 -color FF0000 -force` would remove any other parts of the embed, keeping only the new color.
```

Reply:
Don't really like the "However, ...:" construct tbh. Will keep as is unless you have a big objection to this.

Reply:
no objection, but we should definitely include the `-force` flag for the second example (as it is about that flag) to show usage.

Reply:
Missed that completely 🤦. Will add `-force` to the example.

Reply:
remember, children: too much force pushing is not good for you